docs/queries.md

```diff
@@ -131,7 +131,7 @@ db.query(
 });
 ```
 
-`query` takes named parameters as well:
+`query` takes named parameters as well accepting `$*name*` with `*` being any of the following pairs `{}` `[]` `()` `\\` `<>` :
```

Comment:
It doesn't support $\name\, but supports $/name/

Reply:
thanks, adjusted to forward slash

Reply:
Punctuation is off also. A comma is needed after ‘as wel’ and before ‘with’.